.circleci/config.yml

```diff
@@ -0,0 +1,244 @@
+# Check https://circleci.com/docs/2.0/language-php/ for more details
+#
+
+defaults: &defaults
+  docker:
+    # specify the version you desire here (avoid latest except for testing)
+    - image: andrewberry/drupal_tests:0.4.0
+
+    - image: selenium/standalone-chrome-debug:3.14.0-beryllium
+
+    - image: mariadb:10.3
+      environment:
+        MYSQL_ALLOW_EMPTY_PASSWORD: 1
+
+    # Specify service dependencies here if necessary
+    # CircleCI maintains a library of pre-built images
+    # documented at https://circleci.com/docs/2.0/circleci-images/
+    # - image: circleci/mysql:9.4
+
+  # 'checkout' supports a path key, but not on locals where you test with the
+  # circleci CLI tool.
+  # https://discuss.circleci.com/t/bug-circleci-build-command-ignores-checkout-path-config/13004
+  working_directory: /var/www/html/modules/apigee_m10n
+
+# YAML does not support merging of lists. That means we can't have a default
+# 'steps' configuration, though we can have defaults for individual step
+# properties.
+
+# We use the composer.json as a way to determine if we can cache our build.
+restore_cache: &restore_cache
+  keys:
+  - v4-dependencies-{{ checksum "composer.json" }}-{{ checksum "../../composer.json" }}
+  # fallback to using the latest cache if no exact match is found
+  - v4-dependencies-
+
+# If composer.json hasn't changed, restore the Composer cache directory. We
+# don't restore the lock file so we ensure we get updated dependencies.
+save_cache: &save_cache
+  paths:
+    - /root/.composer/cache/files
+  key: v4-dependencies-{{ checksum "composer.json" }}-{{ checksum "../../composer.json" }}
+
+# Install composer dependencies into the workspace to share with all jobs.
+update_dependencies: &update_dependencies
+  <<: *defaults
+  steps:
+    - checkout
+
+    - restore_cache: *restore_cache
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          ./update-dependencies.sh apigee_m10n
+
+    - save_cache: *save_cache
+
+    - persist_to_workspace:
+        root: /var/www/html
+        paths:
+          - .
+
+# Run Drupal unit and kernel tests as one job. This command invokes the test.sh
+# hook.
+unit_kernel_tests: &unit_kernel_tests
+  <<: *defaults
+  steps:
+    - attach_workspace:
+        at: /var/www/html
+
+    - checkout
+
+    - run:
+        working_directory: /var/www/html
+        command: |
+          cp ./modules/apigee_m10n/.circleci/update-dependencies.sh /var/www/html
+          cp ./modules/apigee_m10n/.circleci/RoboFile.php /var/www/html
+          ./test.sh apigee_m10n
+
+    - store_test_results:
+        path: /var/www/html/artifacts/phpunit
+    - store_artifacts:
+        path: /var/www/html/artifacts
+
+# Run Drupal functional tests. This command invokes the test-functional.sh
+# hook.
+functional_tests: &functional_tests
```

Comment:
I'm wondering if WebDriver or the GoutteDriver is being used form functional tests.

Reply:
I found it. It would default to GoutteDriver as it isn't overridden in <https://github.com/deviantintegral/drupal_tests/blob/master/hooks/test-functional.sh>.